styles.css

```diff
@@ -608,3 +608,220 @@ hr {
+  display: none;
+}
+
+/* ====================================================================================================================== */
+
+/* Styling the desktop version */
+
+@media screen and (min-width: 768px) {
+  .header-logo {
+    height: 65px;
+    z-index: 10;
+    position: relative;
+  }
+
+  .image-icon {
+    display: none;
+  }
+
+  .LOGO {
+    font-family: 'Poppins', system-ui, Courier, monospace;
+    color: #6070ff;
+    font-weight: 700;
+    font-size: 18px;
+    margin: 0;
+    padding: 0;
+    padding-left: 5rem;
+  }
+
+  .nav-content {
+    color: #344563;
+    display: flex;
+    justify-content: space-between;
+    font-family: 'Poppins', system-ui, Courier, monospace;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    list-style: none;
+    padding-right: 8rem;
+    gap: 44px;
+  }
+
+  #body-cover {
+    background-image: url('images/Header\ bg.svg');
+    padding: 0;
+    position: relative;
+    height: calc(100vh - 65px);
+  }
+
+  #body-cover .ctn {
+    width: 70%;
+    height: 50%;
+    margin-left: 10%;
+    margin-top: 0;
+    margin-top: 10%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  .ctn .title {
+    margin-bottom: calc(2vw);
+  }
+
+  .ctn .social-buttons {
+    margin-bottom: calc(2vw);
+  }
+
+  #works-container .works {
+    width: 80%;
+    height: calc(20vw + 250px);
+    margin-left: 10%;
+    padding: 0 20px;
+    box-sizing: border-box;
+    flex-direction: row;
+  }
+
+  .image1 {
+    display: none;
+  }
+
+  .image2 {
+    display: none;
+  }
+
+  .image3 {
+    display: none;
+  }
+
+  .image4 {
+    display: none;
+  }
+
+  .works .illustration {
+    margin-top: 2.4%;
+    height: 90%;
+    width: 50%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-image: url('images/Snapshoot\ Portfolio1.svg');
+    background-size: contain;
+    background-repeat: no-repeat;
+  }
+
+  .works2 .illustration {
+    background-image: url('images/Snapshoot\ Portfolio2.svg');
+    background-size: contain;
+    background-repeat: no-repeat;
+  }
+
+  .works3 .illustration {
+    background-image: url('images/Snapshoot\ Portfolio3.svg');
+    background-size: contain;
+    background-repeat: no-repeat;
+  }
+
+  .works4 .illustration {
+    background-image: url('images/Snapshoot\ Portfolio4.svg');
+    background-size: contain;
+    background-repeat: no-repeat;
+  }
+
+  .works .w_ctn {
+    height: 100%;
+    width: 50%;
+    padding: 10px;
+    padding-top: 5%;
+    box-sizing: border-box;
+  }
+
+  .w_ctn .description {
+    display: inline;
+  }
+
+  .description li {
+    display: inline;
+    margin-right: 12px;
+  }
+
+  .canopy {
+    margin-right: 0;
+  }
+
+  .more-info {
+    margin-top: 2rem;
+  }
+
+  .myself {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    height: auto;
+    padding-left: 5rem;
+    padding-right: 5rem;
+  }
```

Comment:
- [ ] In the about section on the figma design, the left block takes 40% of width of the container and the right block takes 60% width. but in your design both blocks takes 50% of width each. kindly consider fixing it as we need to stick to the figma design.
- [ ] Also kindly consider to add the button interactions styling in all the buttons in your web-page. you can apply these interactions by using `:hover and :active` pseudo-classes in hovering or pressed states.
your design
<img width="1098" alt="Screenshot 2023-07-13 at 10 24 00 PM" src="https://github.com/MarkYegon7/Mark-Portfolio/assets/96683843/42c9fe4c-9c4b-4b24-900a-d18847bda2dc">

figma design
<img width="877" alt="Screenshot 2023-07-13 at 10 23 42 PM" src="https://github.com/MarkYegon7/Mark-Portfolio/assets/96683843/e1bdedcb-8551-4812-b401-4e044f2db259">

button styling guidelines
<img width="403" alt="Screenshot 2023-07-13 at 10 33 02 PM" src="https://github.com/MarkYegon7/Mark-Portfolio/assets/96683843/9afe1da6-2009-4312-b0ee-a2ca60231686">

Reply:
Solved both the container section and the button interactions